packages/app-mobile/components/NoteList.tsx

```diff
@@ -91,7 +92,13 @@ class NoteListComponent extends Component<NoteListProps> {
 				keyExtractor={item => item.id}
 			/>;
 		} else {
-			if (!Folder.atLeastOneRealFolderExists(this.props.folders)) {
+			const noteSource = this.props.notesSource ? JSON.parse(this.props.notesSource) : null;
+			const noteSourceFolder = this.props.folders.find(folder => folder.id === noteSource?.parentId);
+
+			if (noteSource?.parentId === getTrashFolderId() || itemIsInTrash(noteSourceFolder)) {
+				const noItemMessage = _('There are no notes here.');
+				return <Text style={this.styles().noItemMessage}>{noItemMessage}</Text>;
+			} else if (!Folder.atLeastOneRealFolderExists(this.props.folders)) {
```

Comment:
That NoteList component is not even in use. I'm closing this pull request because it doesn't seem like you tested your changes